graphql/schema/index.js

```diff
@@ -1,44 +1,85 @@
-const {buildSchema} = require('graphql');
+
+const { buildSchema } = require("graphql");
 
 module.exports = buildSchema(`
 type category {
   categoryName: String!
   _id: String!
   topicIds: [String!]!
-  idName: String!
+}
+type user {
+  _id: String!
+  email: String!
```

Comment:
you can make email as unique property to restrict user spamming through a single email

Reply:
Okay